tox.ini

```diff
@@ -37,10 +37,10 @@ setenv =
     dev: MAILCHIMP_USER_ACTIONS_SUBACCOUNT = {env:MAILCHIMP_USER_ACTIONS_SUBACCOUNT:devdata}
     tests: ELASTICSEARCH_INDEX = {env:ELASTICSEARCH_INDEX:hypothesis-tests}
     functests: ELASTICSEARCH_INDEX = {env:ELASTICSEARCH_INDEX:hypothesis-functests}
+    functests: PYTEST_ADDOPTS = {env:PYTEST_ADDOPTS:--reverse}
```

Comment:
So the `PYTEST_ADDOPTS` that will be used for the functests will be whatever the user has set in their shell or, if it's not set, will default to `--reverse`.

We should probably add the same for the unittests as well.